### PR TITLE
Peters development

### PR DESCRIPTION
**Pull request by eoa0026**
Changes Index.html to the requirements from the walkthrough, changes style.css to style portfolio, and adds an image folder for all the images used in the portfolio.

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -17,7 +17,7 @@
  */
 function addRandomGreeting() {
   const greetings =
```
Comment (andy-keene) [inline portfolio/src/main/webapp/script.js:19]: I'd like to request we add 

> "I'll allow it." - [Señor Chang](https://media.giphy.com/media/SmoCFhZCi1kzu/giphy.gif)

Comment (andy-keene) [inline portfolio/src/main/webapp/script.js:19]: I'd like to request we add 

> "I'll allow it." - [Señor Chang](https://media.giphy.com/media/SmoCFhZCi1kzu/giphy.gif)

These quotes cracked me up during this code review 🤣

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -17,7 +17,7 @@
...
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+      ['Dont eat the crab dip, YA YA!', 'Streets Ahead', 'I regret nothing', 'Never listen to Pierce.', 'Its called chemistry, I have it with everybody!', 'Cool cool cool.', 'A passing grade? Like a C? Why dont I just get pregnant at a bus station?', 'I give this year a "D", for delightful!'];
```
Comment (andy-keene) [inline portfolio/src/main/webapp/script.js:20]: nit: I think it's "Cool. Cool Cool Cool." [source](https://www.youtube.com/watch?v=8IIrf_JSuQk) 👍

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -3,7 +3,40 @@
...
   width: 650px;
 }
+body {
+  background-color: white;
+}
 
 #greeting-container {
   margin-top: 20px;
 }
+
```
Comment (andy-keene) [inline portfolio/src/main/webapp/style.css:13]: super nit: Try to be consistent with your whitespaces / newlines. In general, regardless of style, consistency is key. Some of these have new lines between, some don't.

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -8,12 +8,28 @@
...
+      <h1>Peter's Portfolio</h1>
+      <p>This is my portfolio. A page to get a breif description of me</p>
+      <img src="image/profile.PNG"/>
+      <h2>Summary</h2>
+      <p>Hello, I am peter, I am a Junior at Cal State Sacramento and I have been coding for about a year now, I am from Texas but born in Nigeria and an international student and I really enjoy building software.</p>
+      <p>I really enjoy coding particularly in java and python, I really like the idea of being able to bring an idea to real life through writing code.  love photography and I do some of that in my free time. I have included some links to my LinkedIn and GitHub profiles and some images that tell you a bit more about me</p>
+      <img src="image/beau.jpg"/>
+      <p style="text-align:center">This is Beau, he loves water for some reason.</p>
+      <img src="image/pink1.jpg"/>
+      <p style="text-align:center">A nice picture of some Japanese cherry Trees I took.</p>
```
Comment (andy-keene) [inline portfolio/src/main/webapp/index.html:20]: **No need to do this!**, but if it wasn't covered in the assignment I want to point out that you can make your own CSS classes, then tag each caption with the class. For example:

```
<div>
  <img src="image/pink1.jpg"/>
  <p class="caption"> A nice picture of some Japanese cherry Trees I took.</p>
```

Later in your CSS file

```
p.caption {
  text-align: center;
}
```

**Why do this?** In-line style (e.g. `style="text-align:center`) is great and works well. But if you find yourself applying it over and over again it's worth asking whether you think you'll want to change it in the future.

Let's say in the future you wanted to make your captions italicized. 

With in-line style you'll need to go through the whole HTML document and update every caption `<p>`  with `<p style="text-align:center;font-style:italic">`. With just a few pictures this is no big deal, but imagine if you had dozens all over the web page!

With using a special CSS class all you would need to do is update your class to be:

```
p.caption {
  text-align: center;
  font-style: italic; // 1 line change!
}
```

**Review by andy-keene — APPROVED** [review]
Awesome! 🍰 

Approved since my comments are mostly nits and shouldn't block this from being (nice work!).

For reference:
* "nit:" is short for nit-pick. It means you don't _have to_ fix these things, but we'd like you to. Generally used for stylistic suggestions.
* "optional:" is kind of self describing. This is typically used for suggestions by the reviewer that aren't required to check the code in. An example would be like "optional: prefer using insertion sort over merge-sort for very small lists"


One other thing, a lot of these images aren't used in the web page. I'd recommend removing them. And so you know, they are publicly visible see: http://pabah-sps-summer21.appspot.com/image/

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (eoa0026) [reply portfolio/src/main/webapp/script.js:19]: will definitely add that 😂

File: portfolio/src/main/webapp/index.html (hunk 4 above)
Comment (eoa0026) [reply portfolio/src/main/webapp/index.html:20]: Neat, that makes it a bit more fluid

**Pull request merged**